assets/launcher/lang/sklmessages_en_US.properties

```diff
@@ -1,22 +1,28 @@
 lang.name=English
 lang.code=en_US
 
+general.issues=Report an issue
+general.support=Support us on Ko-fi
+general.notready=This feature is not ready yet!
+
 tab.news=Update News
 tab.log=Launcher Log
 tab.output=Game Output (%s)
 tab.crash=Crash Report (%s)
 
 sidebar.switch=Switch User
+sidebar.launch=Launch Profile
 sidebar.new=New Profile
 sidebar.edit=Edit Profile
 sidebar.browse=Open Profile Directory
 sidebar.delete=Delete Profile
-sidebar.profiles=Installed Profiles
+sidebar.profilelist=Profiles
 sidebar.loading=Loading Profiles...
 sidebar.settings=Launcher Settings
 sidebar.player=Player
 sidebar.guest=Welcome, guest! Please log in.
 sidebar.versions=Loading versions...
+sidebar.version.latest=Latest Release
```

Comment:
Didn't you mean "Latest Version" here, like in `editor.latest`?

Also, while you are working on an overhauled UI, it sounds like a good time to do something with string `settings.lang.help` and `settings.lang.link`. Maybe the idea from [#71](https://github.com/skmedix/SKlauncher/pull/71#discussion_r205518956)?

Reply:
Yes, it's same as `editor.latest`. But it's a different category so, I has to add it again.

Also thanks for reminding me #71